tests/generate/test_integration_vllm.py

```diff
@@ -1,239 +0,0 @@
-    assert isinstance(sequence, str)
-    assert sequence.find("news") == -1
-
-
-def test_vllm_regex(model):
-    prompt = "Write an email address: "
-    regex_str = r"([a-z]{10})@([a-z]{5})\.([a-z]{3})"
-    generator = generate.regex(model, regex_str)
-
-    # One prompt
-    sequence = generator(prompts=prompt)
-    assert isinstance(sequence, str)
-    assert re.fullmatch(pattern=regex_str, string=sequence) is not None
-
-
-def test_vllm_integer(model):
-    prompt = "Give me an integer: "
-    sequence = generate.format(model, int)(prompt, max_tokens=10)
-    assert isinstance(sequence, int)
-    assert sequence != ""
-    int(sequence)
-
-
-def test_vllm_float(model):
-    prompt = "Give me a floating-point number: "
-    sequence = generate.format(model, float)(prompt, max_tokens=10)
-    assert isinstance(sequence, float)
-
-    assert sequence != ""
-    float(sequence)
-
-
-def test_vllm_bool(model):
-    prompt = "Is this True or False? "
-    sequence = generate.format(model, bool)(prompt, max_tokens=10)
-    assert isinstance(sequence, bool)
-
-    assert sequence != ""
-    bool(sequence)
-
-
-def test_vllm_date(model):
-    prompt = "What day is it today? "
-    sequence = generate.format(model, datetime.date)(prompt, max_tokens=10)
-    assert isinstance(sequence, datetime.date)
-
-
-def test_vllm_time(model):
-    prompt = "What time is it? "
-    sequence = generate.format(model, datetime.time)(prompt, max_tokens=10)
-    assert isinstance(sequence, datetime.time)
-
-
-def test_vllm_datetime(model):
-    prompt = "What time is it? "
-    sequence = generate.format(model, datetime.datetime)(prompt, max_tokens=20)
-    assert isinstance(sequence, datetime.datetime)
-
-
-def test_vllm_choice(model):
-    prompt = "Which one between 'test' and 'choice'? "
-    sequence = generate.choice(model, ["test", "choice"])(prompt)
-    assert sequence == "test" or sequence == "choice"
-
-
-def test_vllm_json_basic(model):
-    prompt = "Output some JSON. "
-
-    class Spam(BaseModel):
-        spam: constr(max_length=10)
-        fuzz: bool
-
-    sampling_params = SamplingParams(temperature=0)
-    result = generate.json(model, Spam, whitespace_pattern="")(
-        prompt, max_tokens=100, seed=1, sampling_params=sampling_params
-    )
-    assert isinstance(result, BaseModel)
-    assert isinstance(result.spam, str)
-    assert isinstance(result.fuzz, bool)
-    assert len(result.spam) <= 10
-
-
-def test_vllm_json_schema(model):
-    prompt = "Output some JSON. "
-
-    schema = """{
-      "title": "spam",
-      "type": "object",
-      "properties": {
-           "foo" : {"type": "boolean"},
-           "bar": {"type": "string", "maxLength": 4}
-        },
-      "required": ["foo", "bar"]
-      }
-    """
-
-    sampling_params = SamplingParams(temperature=0)
-    result = generate.json(model, schema, whitespace_pattern="")(
-        prompt, max_tokens=100, seed=10, sampling_params=sampling_params
-    )
```

Comment:
Why are we removing the `vLLM` integration tests in this PR?

Reply:
This was a mistake, reverted.